nginx/MASZ/src/app/components/automod/automod-card/automod-card.component.ts

```diff
@@ -18,7 +18,8 @@ export class AutomodCardComponent implements OnInit {
     3: 'description',
     4: 'attach_file',
     5: 'history',
-    6: 'do_not_disturb'
+    6: 'do_not_disturb',
+    7: 'history'
```

Comment:
history is already used.
maybe use [`manage_search`](https://fonts.google.com/icons?selected=Material+Icons:gavel&icon.query=manage_search) 
![grafik](https://user-images.githubusercontent.com/24491035/128643219-7a0d37c7-f471-4adc-8c06-3cd1b822c9ff.png)